diff -Nurb pdftohtml-0.36.orig/goo/Makefile pdftohtml-0.36.new/goo/Makefile
--- pdftohtml-0.36.orig/goo/Makefile	2003-06-25 00:41:27.000000000 +0200
+++ pdftohtml-0.36.new/goo/Makefile	2004-12-28 03:23:06.565945520 +0100
@@ -7,13 +7,15 @@
 #
 #========================================================================
 
-SHELL = /bin/sh
+SHELL ?= /bin/sh
 
 srcdir = .
 
-CFLAGS += $(DEBUG) -O2 -DHAVE_CONFIG_H -I.. -I$(srcdir) 
+CFLAGS ?= -O2
+override CFLAGS += $(DEBUG) -DHAVE_CONFIG_H -I.. -I$(srcdir) 
 #-g -DDEBUG_MEM
-CXXFLAGS += $(DEBUG) -O2 -DHAVE_CONFIG_H -I.. -I$(srcdir) 
+CXXFLAGS ?= -O2
+override CXXFLAGS += $(DEBUG) -DHAVE_CONFIG_H -I.. -I$(srcdir) 
 #-g -DDEBUG_MEM
 
 CC ?= cc
--- pdftohtml-0.38/Makefile.orig	2006-01-07 18:33:33.000000000 +0100
+++ pdftohtml-0.38/Makefile	2006-03-23 19:03:34.617833750 +0100
@@ -6,12 +6,12 @@
 #
 #========================================================================
 
-SHELL = /bin/sh
+SHELL ?= /bin/sh
 
 DESTDIR =
 
-prefix = /usr/local
-exec_prefix = ${prefix}
+prefix ?= /usr/local
+exec_prefix ?= ${prefix}
 srcdir = .
 
 INSTALL = /usr/bin/install -c
--- pdftohtml-0.38/src/Makefile.orig	2005-12-21 00:25:16.000000000 +0100
+++ pdftohtml-0.38/src/Makefile	2006-03-23 19:04:29.733278250 +0100
@@ -1,5 +1,5 @@
 
-SHELL = /bin/sh
+SHELL ?= /bin/sh
 
 SRCDIR = .
 XPDFSRCDIR = ../xpdf
@@ -11,7 +11,8 @@
 SPLASHSRCDIR = ../splash
 SPLASHLIBDIR = ../splash
 
-CXXFLAGS = -g -O2 -DHAVE_CONFIG_H -DHAVE_DIRENT_H=1  -I.. -DHAVE_REWINDDIR=1 -DHAVE_POPEN=1 -I.. -I$(GOOSRCDIR) -I$(XPDFSRCDIR) -I$(FOFISRCDIR) -I$(SPLASHSRCDIR) -I$(srcdir)           -I/usr/X11R6/include
+CXXFLAGS ?= -g -O2 -I/usr/X11R6/include
+override CXXFLAGS += -DHAVE_CONFIG_H -DHAVE_DIRENT_H=1  -I.. -DHAVE_REWINDDIR=1 -DHAVE_POPEN=1 -I.. -I$(GOOSRCDIR) -I$(XPDFSRCDIR) -I$(FOFISRCDIR) -I$(SPLASHSRCDIR) -I$(SRCDIR)
 
 LDFLAGS = 
 
--- pdftohtml-0.38/xpdf/Makefile.orig	2005-12-21 00:51:22.000000000 +0100
+++ pdftohtml-0.38/xpdf/Makefile	2006-03-23 19:08:36.748715750 +0100
@@ -18,7 +18,8 @@
 SPLASHSRCDIR = $(srcdir)/../splash
 SPLASHLIBDIR = ../splash
 
-CXXFLAGS +=  -g -O2 -DHAVE_CONFIG_H -I.. -I$(GOOSRCDIR) -I$(FOFISRCDIR) -I$(SPLASHSRCDIR) -I$(srcdir)           -I/usr/X11R6/include
+CXXFLAGS =  -g -O2 -I/usr/X11R6/include
+override CXXFLAGS += -DHAVE_CONFIG_H -I.. -I$(GOOSRCDIR) -I$(FOFISRCDIR) -I$(SPLASHSRCDIR) -I$(srcdir)
 
 CC ?= cc
 CXX ?= c++
--- pdftohtml-0.38/fofi/Makefile.orig	2006-01-07 17:52:54.000000000 +0100
+++ pdftohtml-0.38/fofi/Makefile	2006-03-23 19:13:04.661459250 +0100
@@ -14,9 +14,10 @@
 GOOSRCDIR = $(srcdir)/../goo
 GOOLIBDIR = ../goo
 
-CXXFLAGS = -g -O2 -DHAVE_CONFIG_H -I.. -I$(GOOSRCDIR) -I$(srcdir)
+CXXFLAGS ?= -g -O2
+override CXXFLAGS += -DHAVE_CONFIG_H -I.. -I$(GOOSRCDIR) -I$(srcdir)
 
-CXX = g++
+CXX ?= g++
 AR = ar rc
 RANLIB = ranlib
 
--- pdftohtml-0.38/splash/Makefile.orig	2005-12-20 12:52:56.000000000 +0100
+++ pdftohtml-0.38/splash/Makefile	2006-03-23 19:18:10.596579000 +0100
@@ -6,7 +6,7 @@
 #
 #========================================================================
 
-SHELL = /bin/sh
+SHELL ?= /bin/sh
 
 srcdir = .
 
@@ -16,9 +16,10 @@
 FOFISRCDIR = $(srcdir)/../fofi
 FOFILIBDIR = ../fofi
 
-CXXFLAGS = -g -O2 -DHAVE_CONFIG_H -I.. -I$(GOOSRCDIR) -I$(FOFISRCDIR) -I$(srcdir)  
+CXXFLAGS ?= -g -O2
+override CXXFLAGS += -DHAVE_CONFIG_H -I.. -I$(GOOSRCDIR) -I$(FOFISRCDIR) -I$(srcdir)  
 
-CXX = g++
+CXX ?= g++
 AR = ar rc
 RANLIB = ranlib
 
