--- pdftohtml_0_32b.orig/pdftohtml	Sun Mar 17 09:38:05 2002
+++ pdftohtml_0_32b/pdftohtml	Mon Apr  1 21:47:53 2002
@@ -1,12 +1,4 @@
-#!/usr/local/bin/bash
-
-#### User modifiable options
-
-PDFTOHTMLBIN_DIR=$HOME/xpdf/work/pdftohtml
-RM=/bin/rm
-
-#### End of User modifiable options
-
+#!/bin/bash
 
 function printhelp (){
 echo "pdftohtml version 0.32b
@@ -85,7 +77,7 @@
   mkdir -p $DIR
 fi
 
-$PDFTOHTMLBIN_DIR/pdftohtml.bin $FRAMES $STDOUT $FIRST $LAST $EXT $QUIET $CSS $IGNOREIMAGES $SCALE $ENC $XML $PDFFILE $DIR/$TARGET
+pdftohtml $FRAMES $STDOUT $FIRST $LAST $EXT $QUIET $CSS $IGNOREIMAGES $SCALE $ENC $XML $PDFFILE $DIR/$TARGET
 
 
 image=$DIR/image.log
@@ -109,7 +101,7 @@
     PAPER_ROTATE=`grep "ROTATE" $image`
     PAPER_ROTATE=`echo $PAPER_ROTATE|cut -f2 -d'='`
     
-    $PDFTOHTMLBIN_DIR/pdftops.bin $FIRST $LAST -paperw $PAPER_WIDTH -paperh $PAPER_HEIGHT -notext $PDFFILE $DIR/$PSFILE
+    pdftops $FIRST $LAST -paperw $PAPER_WIDTH -paperh $PAPER_HEIGHT -notext $PDFFILE $DIR/$PSFILE
 
     cd $DIR
 
